docs/getting-started.md

```diff
@@ -35,31 +35,31 @@ irma session --server http://localhost:8088 --disclose pbdf.pbdf.irmatube.type
 ([IRMATube attributes](https://privacybydesign.foundation/attribute-index/en/pbdf.pbdf.irmatube.html) are available on the [IRMATube demo](https://privacybydesign.foundation/demo/irmaTube/)) page. This will print a QR that you can scan with your IRMA app, and the attribute contents after they have been received and verified by the server. `irma session` can also perform issuance sessions and attribute-based signature sessions. If you pass  `-v` it logs the session request JSON that it sends to your `irma server`.
 
 
-## Installing `irmajs` and an example webpage
-Download the `irmajs` source code, as [zip](https://github.com/privacybydesign/irmajs/archive/master.zip) from GitHub, or using git:
+## Installing `irma-frontend` and an example webpage
+Download the `irma-frontend-packages` source code, as [zip](https://github.com/privacybydesign/irma-frontend-packages/archive/master.zip) from GitHub, or using git:
 ```shell
-git clone https://github.com/privacybydesign/irmajs && cd irmajs
+git clone https://github.com/privacybydesign/irma-frontend-packages && cd irma-frontend-packages
 ```
 
-Like the `irma` command line tool, `irmajs` needs to be compiled before it can be used in the browser. This bundles all dependencies, images and CSS into one JavaScript file. You can obtain a compiled version of `irmajs` in one of the following two ways:
+Like the `irma` command line tool, the `irma-frontend` bundle package needs to be compiled before it can be used in the browser. This bundles all dependencies, images and CSS into one JavaScript file. You can obtain a compiled version of `irma-frontend` in one of the following two ways:
 
-* **Download prebuilt binary**: From our [CI build server](https://gitlab.science.ru.nl/irma/github-mirrors/irmajs/-/jobs/artifacts/master/download?job=bundle). Extract the files into the `dist` folder of `irmajs.`
+* **Download prebuilt binary**: From our [CI build server](https://gitlab.science.ru.nl/irma/github-mirrors/irma-frontend-packages/-/jobs/artifacts/master/browse/irma-frontend?job=irma-frontend) or from the [NPM repository](https://www.npmjs.com/package/@privacybydesign/irma-frontend).
 * **Compile from source**, as follows:
   ```shell
-  git clone https://github.com/privacybydesign/irmajs && cd irmajs
-  npm i
-  npm run build
+  git clone https://github.com/privacybydesign/irma-frontend-packages && cd irma-frontend
+  ./build.sh
```

Comment:
`irma-frontend-packages` uses npm for all modules and examples, so in principle doing an `npm install` (and in the webpack modules an additional `npm run release`) in the example or module you want to build would be enough. Are these instructions still of added value then?